apps/api/src/app/auth/auth.controller.ts

```diff
@@ -83,7 +83,7 @@ export class AuthController {
     logOut(@Req() req: Request, @Res() res: Response) {
         res.cookie("jwt", "", {
             httpOnly: true,
-            maxAge: 0
+            expires: new Date()
```

Comment:
What does that mean?

Reply:
`expires =  current-date` would mean cookie should expire now.
But maxAge:0 should also do the same, but it was not working. This library converts the maxAge milliseconds to expires (Date) internally. I didn't debug in to why it was not working as expected, but setting expires directly likes this work well.